main.py

```diff
@@ -0,0 +1,248 @@
+import random
+
+suits = ('Hearts', 'Diamonds', 'Spades', 'Clubs')
+ranks = ('Two', 'Three', 'Four', 'Five', 'Six', 'Seven', 'Eight', 'Nine', 'Ten', 'Jack', 'Queen', 'King', 'Ace')
+values = {'Two':2, 'Three':3, 'Four':4, 'Five':5, 'Six':6, 'Seven':7, 'Eight':8, 
+            'Nine':9, 'Ten':10, 'Jack':11, 'Queen':12, 'King':13, 'Ace':14}
+playing = True
+
+class Card():
+    
+    def __init__(self,suit,rank):
+        self.suit = suit
+        self.rank = rank
+            
+    def __str__(self):
+        return self.rank + " of " + self.suit
+    
+    
+class Deck():
+    
+    def __init__(self):
+        self.deck = [] 
+        for suit in suits:
```

Comment:
Where do suit and rank variables come from? What's gonna happen when You will extract the class to a different file? Classes should not depend on local variables and be re-useable.